x-pack/legacy/plugins/lens/server/routes/index_stats.ts

```diff
@@ -95,6 +96,116 @@ export async function initStatsRoute(setup: CoreSetup) {
       }
     }
   );
+
+  router.post(
+    {
+      path: '/index_stats/{indexPatternTitle}/field',
+      validate: {
+        params: schema.object({
+          indexPatternTitle: schema.string(),
+        }),
+        body: schema.object(
+          {
+            // query: schema.object({}),
+            earliest: schema.string(),
```

Comment:
I find earliest / latest to be a bit confusing. Not sure what we should use, but maybe just keep the date range convention used elsewhere: `fromDate` and `toDate`. Also, should the schema check to ensure these are dates, or should we just let Elasticsearch handle that for us?